scripts/benchmark_k.py

```diff
@@ -4,9 +4,11 @@
 import anndata
 import pandas as pd
 import gc
+import scanpy as sc
 
-sys.path.insert(1, './src/')
-from utils import *
+from markermap.utils import SmashPyWrapper, LassoNetWrapper, RandomBaseline
+from markermap.utils import MarkerMap, ConcreteVAE_NMSL, VAE_Gumbel_GlobalGate, VAE_l1_diag
+from markermap.utils import benchmark, plot_benchmarks
```

Comment:
Later, I am planning on splitting utils into 3 files: utils.py, vae_models.py, which will include that whole family of models, and then something like other_models.py which will include the SmashPyWrapper, LassoNetWrapper, and RandomBaseline and any other models that we add like that.